base_layer/core/src/transactions/aggregated_body.rs

```diff
@@ -294,6 +294,7 @@ impl AggregateBody {
         coinbase_min_maturity: u64,
         factories: &CryptoFactories,
         height: u64,
+        maximum_coinbase_amount: u64,
```

Comment:
Amount suggests monetary value
```suggestion
        maximum_coinbase_count: u64,
```